### PR TITLE
on exception print which query set fails

### PR DESCRIPTION
**Pull request by dothebart**
This test iterates over a set of test-query/bind values. 
In case of failure it doesn't output any information where it fails. 
This PR will fix this.

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7376/

File: tests/js/server/shell/shell-cluster-gatherblocks.js [hunk 1]
```diff
@@ -156,10 +155,19 @@ function gatherBlocksTestSuite() {
...
             for(var i=0; i < loop; i++){
               let start = Date.now();
-              rv = db._query(t.query , bindvars);
-              rv = rv.toArray().map(doc => { return doc.value; } );
-              time += (Date.now() - start);
+              try  {
+                rv = db._query(t.query , bindvars);
+                rv = rv.toArray().map(doc => { return doc.value; } );
+                time += (Date.now() - start);
+              }
```
Comment (jsteemann) [inline tests/js/server/shell/shell-cluster-gatherblocks.js]: please add `fail()` here

File: tests/js/server/shell/shell-cluster-gatherblocks.js [hunk 2]
```diff
@@ -156,10 +155,19 @@ function gatherBlocksTestSuite() {
...
+              catch (ex) {
+                print("Failed in " + t.query);
+                print(bindvars);
+                print(db._explain(t.query , bindvars));
```
Comment (jsteemann) [inline tests/js/server/shell/shell-cluster-gatherblocks.js]: `db._explain` will print by itself, so its result does not need to be printed.

**Comment by jsteemann** [comment]
Tests blue

**Pull request merged**